ui/src/pages/supplyInvitations/SupplyInvitations.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from "react";
+import Contracts from "../../components/Contracts/Contracts";
+import { useStreamQuery, useParty, useLedger } from "@daml/react";
+import { QuoteRequestSupplyInvitation }
+  from "@daml.js/supplychain-1.0.0/lib/DA/RefApps/SupplyChain/QuoteRequest";
+import { useState } from "react";
+import { CreateEvent } from "@daml/ledger";
+import { OrderedProductList } from "../quoteRequests/OrderedProductList";
+
+export default function SupplyInvitations() {
+
+  const party = useParty();
+  const ledger = useLedger();
+  const roles =
+    useStreamQuery(QuoteRequestSupplyInvitation);
```

Comment:
Again, ignore if this is done by a formatter.
```suggestion
  const roles = useStreamQuery(QuoteRequestSupplyInvitation);
```